src/util/RouterUtil.js

```diff
@@ -100,8 +100,13 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
     // Token has expired - no longer valid. Navigate back to primary auth.
     if (err.errorCode === ErrorCodes.INVALID_TOKEN_EXCEPTION) {
       router.appState.set('flashError', err);
-      router.controller.state.set('navigateDir', Enums.DIRECTION_BACK);
-      router.navigate('', { trigger: true });
+      if (router.controller) {
```

Comment:
do we still need this change after leveraging `introspecSuccess/Error`?

Reply:
removed